packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -1282,18 +1277,18 @@ class ObjectGroup extends ObjectGroupBase {
   }
 
   Future<bool> handleSetSelectionDaemon(
-    Future<Object> setSelectionResult,
+    Future<Object?> setSelectionResult,
     bool uiAlreadyUpdated,
   ) async {
     if (disposed) return false;
     // TODO(jacobr): we need to cancel if another inspect request comes in while we are trying this one.
     final json = await setSelectionResult;
```

Comment:
rename json to selectionChanged so that is clearer what this value is

Reply:
done